ads/google/a4a/test/test-google-ads-a4a-config.js

```diff
@@ -156,7 +156,7 @@ describe('a4a_config', () => {
   it('should attach expt ID and return true when expt is on', () => {
     rand.returns(0.75);  // Random value to select the 2nd branch
     expect(googleAdsIsA4AEnabled(win, element, EXP_ID,
-        EXTERNAL_BRANCHES, INTERNAL_BRANCHES),
+        EXTERNAL_BRANCHES, INTERNAL_BRANCHES, null),
```

Comment:
Also part of why I set it to null was that I wasn't sure if this was DoubleClick or AdSense, and made a note to investigate that in a bit. But I see from lines 40-47 that it's neither, the EIDs are not real. So in that case I'll set MEASURE_ON_RENDER=5 and use that instead of null.